haskellPackages: update stackage and hackage

expipiplus1 opened the pull request:
### This Merge

This PR is the regular merge of the `haskell-updates` branch into `master`.

This branch is being continually built and tested by hydra at https://hydra.nixos.org/jobset/nixpkgs/haskell-updates. You may be able to find an up-to-date Hydra build report at [cdepillabout/nix-haskell-updates-status](https://github.com/cdepillabout/nix-haskell-updates-status).

We roughly aim to merge these `haskell-updates` PRs at least once every two weeks. See the @NixOS/haskell [team calendar](https://cloud.maralorn.de/apps/calendar/p/H6migHmKX7xHoTFa) for who is currently in charge of this branch.

### haskellPackages Workflow Summary

Our workflow is currently described in [`pkgs/development/haskell-modules/HACKING.md`](https://github.com/NixOS/nixpkgs/blob/haskell-updates/pkgs/development/haskell-modules/HACKING.md).

The short version is this:
* We regularly update the Stackage and Hackage pins on `haskell-updates` (normally at the beginning of a merge window).
* The community fixes builds of Haskell packages on that branch.
* We aim at at least one merge of `haskell-updates` into `master` every two weeks.
* We only do the merge if the [`mergeable`](https://hydra.nixos.org/job/nixpkgs/haskell-updates/mergeable) job is succeeding on hydra.
* If a [`maintained`](https://hydra.nixos.org/job/nixpkgs/haskell-updates/maintained) package is still broken at the time of merge, we will only merge if the maintainer has been pinged 7 days in advance. (If you care about a Haskell package, become a maintainer!)

---

This is the follow-up to #188758. Come to [#haskell:nixos.org](https://matrix.to/#/#haskell:nixos.org) if you have any questions.

expipiplus1 committed:
haskellPackages: stackage LTS 19.20 -> LTS 19.22

This commit has been generated by maintainers/scripts/haskell/update-stackage.sh

expipiplus1 committed:
all-cabal-hashes: 2022-08-28T23:15:42Z -> 2022-09-11T02:31:18Z

This commit has been generated by maintainers/scripts/haskell/update-hackage.sh

expipiplus1 committed:
haskellPackages: regenerate package set based on current config

This commit has been generated by maintainers/scripts/haskell/regenerate-hackage-packages.sh

cdepillabout committed:
haskellPackages.hspec_2_10_5: bump

github-actions[bot] committed:
Merge master into haskell-updates

github-actions[bot] committed:
Merge master into haskell-updates

arcz committed:
haskellPackages.hevm: add maintainer

cdepillabout committed:
Merge pull request #191046 from arcz/hevm-maintainer

haskellPackages.hevm: add maintainer

cdepillabout committed:
haskellPackages: regenerate package set based on current config

This commit has been generated by maintainers/scripts/haskell/regenerate-hackage-packages.sh

maralorn commented:
current issues seem mainly to be megaparsec on ghc < 9.0 and the new xmonad release. I will post the report later, when it contains less timeouts.

github-actions[bot] committed:
Merge master into haskell-updates

sternenseemann commented:
### [haskell-updates build report from hydra](https://hydra.nixos.org/jobset/nixpkgs/haskell-updates)
*evaluation [1780628](https://hydra.nixos.org/eval/1780628) of nixpkgs commit [ec97ea8](https://github.com/NixOS/nixpkgs/commits/ec97ea81fae7801a2165f878236963f15d3dd19c) as of 2022-09-14 06:50 UTC*

:red_circle: **Branch not mergeable**
  * `mergeable` jobset failed.
  * `maintained` jobset failed.

#### Build summary

 | Platform | Failed :x: | DependencyFailed :heavy_exclamation_mark: | TimedOut :hourglass::no_entry_sign: | Success :heavy_check_mark: | 
 | --- | --- | --- | --- | --- | 
 | [aarch64-linux :iphone:](https://hydra.nixos.org/eval/1780628?filter=.aarch64-linux) | 32 | 32 | 4 | 6617 | 
 | [x86_64-darwin :apple:](https://hydra.nixos.org/eval/1780628?filter=.x86_64-darwin) | 64 | 11 | 8 | 6546 | 
 | [x86_64-linux :penguin:](https://hydra.nixos.org/eval/1780628?filter=.x86_64-linux) | 11 | 13 | 4 | 6701 | 
#### Maintained packages with build failure
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190503193) [[:apple::x:]](https://hydra.nixos.org/build/190503204) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190503212) [hedgewars](https://hydra.nixos.org/eval/1780628?filter=hedgewars) @fpletz @kragniz
- [ ] [spacecookie](https://hydra.nixos.org/eval/1780628?filter=spacecookie) @sternenseemann
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190435632) [[:apple::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/190421059) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190421317) [toplevel](https://hydra.nixos.org/eval/1780628?filter=spacecookie)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190421398) [[:apple::x:]](https://hydra.nixos.org/build/190434095) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190424903) [haskellPackages](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.spacecookie)
- [ ] [xmonad](https://hydra.nixos.org/eval/1780628?filter=xmonad) @NeQuissimus @ivanbrennan @peti
  - [[:iphone::x:]](https://hydra.nixos.org/build/190435289) [[:apple::x:]](https://hydra.nixos.org/build/190423052) [[:penguin::x:]](https://hydra.nixos.org/build/190438027) [haskellPackages](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.xmonad)
  -   [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190503218) [nixosTests](https://hydra.nixos.org/eval/1780628?filter=nixosTests.xmonad)
#### Maintained packages with failed dependency
- [ ] [haskell-language-server](https://hydra.nixos.org/eval/1780628?filter=haskell-language-server) @maralorn
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190429450) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190425373) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190424293) [toplevel](https://hydra.nixos.org/eval/1780628?filter=haskell-language-server)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190434864) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190431932) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190425146) [haskell.packages.ghc8107](https://hydra.nixos.org/eval/1780628?filter=haskell.packages.ghc8107.haskell-language-server)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190418417) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190423241) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190436674) [haskell.packages.ghc884](https://hydra.nixos.org/eval/1780628?filter=haskell.packages.ghc884.haskell-language-server)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190422826) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190427158) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190434052) [haskell.packages.ghc902](https://hydra.nixos.org/eval/1780628?filter=haskell.packages.ghc902.haskell-language-server)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190422274) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190434720) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190433597) [haskell.packages.ghc924](https://hydra.nixos.org/eval/1780628?filter=haskell.packages.ghc924.haskell-language-server)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190436858) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190426855) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190434471) [haskellPackages](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.haskell-language-server)
- [ ] [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190440239) [maintained](https://hydra.nixos.org/eval/1780628?filter=maintained) @cdepillabout @expipiplus1 @maralorn @sternenseemann
- [ ] [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190503188) [mergeable](https://hydra.nixos.org/eval/1780628?filter=mergeable) @cdepillabout @expipiplus1 @maralorn @sternenseemann
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190428675) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190423532) [taffybar](https://hydra.nixos.org/eval/1780628?filter=taffybar) @rvl
- [ ] [weeder](https://hydra.nixos.org/eval/1780628?filter=weeder) @maralorn
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190420317) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190432075) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190433887) [haskell.packages.ghc8107](https://hydra.nixos.org/eval/1780628?filter=haskell.packages.ghc8107.weeder)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190421693) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190432200) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190429384) [haskell.packages.ghc902](https://hydra.nixos.org/eval/1780628?filter=haskell.packages.ghc902.weeder)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190434980) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190423387) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190429807) [haskell.packages.ghc924](https://hydra.nixos.org/eval/1780628?filter=haskell.packages.ghc924.weeder)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190431343) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190435592) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190422413) [haskellPackages](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.weeder)
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190433806) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190429363) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190430714) [haskellPackages.xmonad-contrib](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.xmonad-contrib) @peti
- [ ] [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190503208) [nixosTests.xmonad-xdg-autostart](https://hydra.nixos.org/eval/1780628?filter=nixosTests.xmonad-xdg-autostart) @oxalica
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190433458) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190420962) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190433938) [xmonadctl](https://hydra.nixos.org/eval/1780628?filter=xmonadctl) @ajgrf
#### Unmaintained packages with build failure
<details><summary>86 job(s) </summary>

- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190431005) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190426145) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190421465) [haskellPackages.OrderedBits](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.OrderedBits)  :arrow_heading_up: 5 | 36
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190418284) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190435872) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190427652) [haskellPackages.hw-json-simd](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hw-json-simd)  :arrow_heading_up: 4 | 8
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190431426) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190435561) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190437758) [haskellPackages.hw-simd](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hw-simd)  :arrow_heading_up: 4 | 8
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190426554) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190433701) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190435927) [haskellPackages.long-double](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.long-double)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190437610) [[:apple::x:]](https://hydra.nixos.org/build/190418485) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190419817) [haskellPackages.quic](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.quic)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190418721) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190424366) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190418471) [haskellPackages.freetype2](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.freetype2)  :arrow_heading_up: 1 | 8
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190426179) [[:apple::x:]](https://hydra.nixos.org/build/190428059) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190423250) [haskellPackages.easytensor](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.easytensor)  :arrow_heading_up: 1 | 1
- [ ] [futhark](https://hydra.nixos.org/eval/1780628?filter=futhark)  :arrow_heading_up: 1 | 1
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190429894) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190435640) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190421580) [toplevel](https://hydra.nixos.org/eval/1780628?filter=futhark)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190433463) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190422015) [[:penguin::x:]](https://hydra.nixos.org/build/190438453) [haskellPackages](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.futhark)
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190424218) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190425062) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190431214) [haskellPackages.haskell-admin-health](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.haskell-admin-health)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190420884) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190436438) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190425960) [haskellPackages.nlopt-haskell](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.nlopt-haskell)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190421787) [[:apple::x:]](https://hydra.nixos.org/build/190430196) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190438325) [haskellPackages.openal-ffi](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.openal-ffi)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190426560) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190422117) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190422325) [haskellPackages.swisstable](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.swisstable)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190434587) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190427671) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190433104) [haskellPackages.unicode-properties](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.unicode-properties)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190436937) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190419069) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190435488) [haskellPackages.flatparse](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.flatparse)  :arrow_heading_up: 0 | 14
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190428795) [[:apple::x:]](https://hydra.nixos.org/build/190430540) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190429741) [haskellPackages.PyF](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.PyF)  :arrow_heading_up: 0 | 4
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190435452) [[:apple::x:]](https://hydra.nixos.org/build/190423209) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190423465) [haskellPackages.hmidi](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hmidi)  :arrow_heading_up: 0 | 4
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190418264) [[:apple::x:]](https://hydra.nixos.org/build/190426060) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190423481) [haskellPackages.posix-socket](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.posix-socket)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190437004) [[:apple::x:]](https://hydra.nixos.org/build/190422721) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190432406) [haskellPackages.gi-gdkx11](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.gi-gdkx11)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190420703) [[:apple::x:]](https://hydra.nixos.org/build/190437843) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190434958) [haskellPackages.hamid](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hamid)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190427217) [[:apple::x:]](https://hydra.nixos.org/build/190422876) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190434364) [haskellPackages.hmatrix-morpheus](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hmatrix-morpheus)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190435315) [[:apple::x:]](https://hydra.nixos.org/build/190424984) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190422869) [haskellPackages.huckleberry](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.huckleberry)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190422851) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190421788) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190427574) [haskellPackages.picosat](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.picosat)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190421878) [[:apple::x:]](https://hydra.nixos.org/build/190420004) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190419766) [haskellPackages.select](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.select)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190433888) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190420467) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190429972) [haskellPackages.simple-vec3](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.simple-vec3)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190435304) [[:apple::x:]](https://hydra.nixos.org/build/190437845) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190425782) [haskellPackages.sysinfo](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.sysinfo)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190426515) [[:apple::x:]](https://hydra.nixos.org/build/190430439) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190423606) [haskellPackages.FractalArt](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.FractalArt) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190435051) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190434382) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190436557) [haskellPackages.HsASA](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.HsASA) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190418395) [[:apple::x:]](https://hydra.nixos.org/build/190425917) [[:penguin::x:]](https://hydra.nixos.org/build/190429391) [haskellPackages.bidirectional-instances](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.bidirectional-instances) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190429501) [[:apple::x:]](https://hydra.nixos.org/build/190434413) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190436327) [haskellPackages.chiphunk](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.chiphunk) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190426409) [[:apple::x:]](https://hydra.nixos.org/build/190438510) [[:penguin::x:]](https://hydra.nixos.org/build/190437098) [haskellPackages.cicero-api](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.cicero-api) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190434552) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190438494) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190432930) [haskellPackages.comfort-fftw](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.comfort-fftw) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190436349) [[:apple::x:]](https://hydra.nixos.org/build/190429906) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190437140) [haskellPackages.diskhash](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.diskhash) 
- [ ] [elm2nix](https://hydra.nixos.org/eval/1780628?filter=elm2nix) 
  - [[:iphone::x:]](https://hydra.nixos.org/build/190420803) [[:apple::x:]](https://hydra.nixos.org/build/190436375) [[:penguin::x:]](https://hydra.nixos.org/build/190418421) [toplevel](https://hydra.nixos.org/eval/1780628?filter=elm2nix)
  - [[:iphone::x:]](https://hydra.nixos.org/build/190420972) [[:apple::x:]](https://hydra.nixos.org/build/190422235) [[:penguin::x:]](https://hydra.nixos.org/build/190422171) [haskellPackages](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.elm2nix)
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190435792) [[:apple::x:]](https://hydra.nixos.org/build/190436062) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190437953) [haskellPackages.epub-tools](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.epub-tools) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190422331) [[:apple::x:]](https://hydra.nixos.org/build/190421828) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190421042) [haskellPackages.fudgets](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.fudgets) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190433305) [[:apple::x:]](https://hydra.nixos.org/build/190423550) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190436480) [haskellPackages.gerrit](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.gerrit) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190428783) [[:apple::x:]](https://hydra.nixos.org/build/190418276) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190435006) [haskellPackages.ghc-gc-hook](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.ghc-gc-hook) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/190420084) [haskellPackages.gi-gtkosxapplication](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.gi-gtkosxapplication) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190419663) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190430872) [haskellPackages.gnome-keyring](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.gnome-keyring) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/190425865) [haskellPackages.gtk-mac-integration](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.gtk-mac-integration) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190427153) [[:apple::x:]](https://hydra.nixos.org/build/190424726) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190431153) [haskellPackages.gtk-traymanager](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.gtk-traymanager) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/190433684) [haskellPackages.gtk3-mac-integration](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.gtk3-mac-integration) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190428754) [[:apple::x:]](https://hydra.nixos.org/build/190424290) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190420486) [haskellPackages.hid](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hid) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190419758) [[:apple::x:]](https://hydra.nixos.org/build/190418699) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190438647) [haskellPackages.highlight](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.highlight) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190422596) [[:apple::x:]](https://hydra.nixos.org/build/190425427) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190429399) [haskellPackages.hinotify-conduit](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hinotify-conduit) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190437030) [[:apple::x:]](https://hydra.nixos.org/build/190420833) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190435871) [haskellPackages.hsshellscript](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hsshellscript) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190438505) [[:apple::x:]](https://hydra.nixos.org/build/190422004) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190435562) [haskellPackages.hssourceinfo](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hssourceinfo) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190432750) [[:apple::x:]](https://hydra.nixos.org/build/190419419) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190433284) [haskellPackages.interprocess](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.interprocess) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190419382) [[:apple::x:]](https://hydra.nixos.org/build/190427460) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190429506) [haskellPackages.intricacy](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.intricacy) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190424645) [[:apple::x:]](https://hydra.nixos.org/build/190420649) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190423584) [haskellPackages.ipcvar](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.ipcvar) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190429809) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190420867) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190435388) [haskellPackages.jammittools](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.jammittools) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/190429773) [haskellPackages.kqueue](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.kqueue) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190429726) [[:apple::x:]](https://hydra.nixos.org/build/190421244) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190422406) [haskellPackages.linux-framebuffer](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.linux-framebuffer) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190437436) [[:apple::x:]](https://hydra.nixos.org/build/190424746) [[:penguin::x:]](https://hydra.nixos.org/build/190427072) [haskellPackages.loc-test](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.loc-test) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190427512) [[:apple::x:]](https://hydra.nixos.org/build/190437142) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190423839) [haskellPackages.mediawiki2latex](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.mediawiki2latex) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190437330) [[:apple::x:]](https://hydra.nixos.org/build/190420114) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190424042) [haskellPackages.memfd](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.memfd) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190431691) [[:apple::x:]](https://hydra.nixos.org/build/190420349) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190436127) [haskellPackages.mercury-api](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.mercury-api) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190438583) [[:apple::x:]](https://hydra.nixos.org/build/190431381) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190419732) [haskellPackages.nano-cryptr](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.nano-cryptr) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190423899) [[:apple::x:]](https://hydra.nixos.org/build/190420807) [[:penguin::x:]](https://hydra.nixos.org/build/190434935) [haskellPackages.panfiguration](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.panfiguration) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190424244) [[:apple::x:]](https://hydra.nixos.org/build/190420798) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190422508) [haskellPackages.persistent-pagination](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.persistent-pagination) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190433438) [[:apple::x:]](https://hydra.nixos.org/build/190434525) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190420164) [haskellPackages.phatsort](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.phatsort) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190418261) [[:apple::x:]](https://hydra.nixos.org/build/190433230) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190436409) [haskellPackages.ping-wrapper](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.ping-wrapper) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190419667) [[:apple::x:]](https://hydra.nixos.org/build/190428126) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190424911) [haskellPackages.posix-timer](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.posix-timer) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190419559) [[:apple::x:]](https://hydra.nixos.org/build/190418650) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190437706) [haskellPackages.procex](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.procex) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190427015) [[:apple::x:]](https://hydra.nixos.org/build/190426253) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190437704) [haskellPackages.pthread](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.pthread) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190423556) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190432668) [[:penguin::x:]](https://hydra.nixos.org/build/190433161) [haskellPackages.reserve](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.reserve) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190427344) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190428826) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190438377) [haskellPackages.risc386](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.risc386) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190422483) [[:apple::x:]](https://hydra.nixos.org/build/190428959) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190434501) [haskellPackages.sfml-audio](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.sfml-audio) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190433665) [[:apple::x:]](https://hydra.nixos.org/build/190426378) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190437206) [haskellPackages.shared-memory](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.shared-memory) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190423958) [[:apple::x:]](https://hydra.nixos.org/build/190427928) [[:penguin::x:]](https://hydra.nixos.org/build/190433784) [haskellPackages.significant-figures](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.significant-figures) 
- [ ] [[:iphone::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/190418546) [[:apple::x:]](https://hydra.nixos.org/build/190420295) [[:penguin::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/190434739) [haskellPackages.skews](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.skews) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190423315) [[:apple::x:]](https://hydra.nixos.org/build/190429164) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190438488) [haskellPackages.slugify](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.slugify) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190436754) [[:apple::x:]](https://hydra.nixos.org/build/190436467) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190428684) [haskellPackages.streamly-lmdb](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.streamly-lmdb) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190432813) [[:apple::x:]](https://hydra.nixos.org/build/190426355) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190422800) [haskellPackages.tailfile-hinotify](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.tailfile-hinotify) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190421659) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190430013) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190419700) [haskellPackages.wiringPi](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.wiringPi) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190429268) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190428038) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190426739) [haskellPackages.x86-64bit](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.x86-64bit) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190435907) [[:apple::x:]](https://hydra.nixos.org/build/190428069) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190424628) [haskellPackages.xmonad-utils](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.xmonad-utils) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190436162) [[:apple::x:]](https://hydra.nixos.org/build/190419569) [[:penguin::x:]](https://hydra.nixos.org/build/190433564) [haskellPackages.xstatic-th](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.xstatic-th) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190424161) [[:apple::x:]](https://hydra.nixos.org/build/190427020) [[:penguin::x:]](https://hydra.nixos.org/build/190427961) [haskellPackages.yarl](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.yarl) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190426980) [[:apple::x:]](https://hydra.nixos.org/build/190426061) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190420502) [haskellPackages.yoga](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.yoga) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190436342) [[:apple::x:]](https://hydra.nixos.org/build/190436771) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190438349) [haskellPackages.zot](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.zot) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190418762) [[:apple::x:]](https://hydra.nixos.org/build/190433345) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190423003) [haskellPackages.zxcvbn-c](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.zxcvbn-c) 
</details>

#### Unmaintained packages with failed dependency
<details><summary>37 job(s) </summary>

- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190423047) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190428573) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190433396) [haskellPackages.PrimitiveArray](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.PrimitiveArray)  :arrow_heading_up: 4 | 35
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190430547) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190434132) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190433095) [haskellPackages.BiobaseTypes](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.BiobaseTypes)  :arrow_heading_up: 3 | 21
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190426955) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190420300) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190424992) [haskellPackages.hw-json-standard-cursor](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hw-json-standard-cursor)  :arrow_heading_up: 2 | 6
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190420794) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190420979) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190432705) [haskellPackages.hw-json-simple-cursor](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hw-json-simple-cursor)  :arrow_heading_up: 2 | 4
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190424585) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190428752) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190430726) [haskellPackages.BiobaseENA](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.BiobaseENA)  :arrow_heading_up: 1 | 18
- [ ] [hoogle](https://hydra.nixos.org/eval/1780628?filter=hoogle)  :arrow_heading_up: 1 | 3
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190428599) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190421911) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190434925) [haskell.packages.ghc8107](https://hydra.nixos.org/eval/1780628?filter=haskell.packages.ghc8107.hoogle)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190420304) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190423503) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190431705) [haskell.packages.ghc884](https://hydra.nixos.org/eval/1780628?filter=haskell.packages.ghc884.hoogle)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190438524) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190431232) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190433741) [haskell.packages.ghc902](https://hydra.nixos.org/eval/1780628?filter=haskell.packages.ghc902.hoogle)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190427705) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190427424) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190429826) [haskell.packages.ghc924](https://hydra.nixos.org/eval/1780628?filter=haskell.packages.ghc924.hoogle)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190429525) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190430879) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190419759) [haskellPackages](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hoogle)
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190435673) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190419901) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190435481) [haskellPackages.hw-json](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hw-json)  :arrow_heading_up: 1 | 3
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190430941) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190430235) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190426835) [haskellPackages.http3](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.http3)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/190438326) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190423583) [[:penguin::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/190420043) [haskellPackages.wss-client](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.wss-client)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190427282) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190435264) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190435122) [haskellPackages.BiobaseXNA](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.BiobaseXNA)  :arrow_heading_up: 0 | 17
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190433132) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190420995) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190433491) [haskellPackages.BiobaseFasta](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.BiobaseFasta)  :arrow_heading_up: 0 | 3
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190418728) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190430015) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190426945) [haskellPackages.hw-dsv](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hw-dsv)  :arrow_heading_up: 0 | 3
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190430393) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190421592) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190438165) [haskellPackages.hw-json-lens](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hw-json-lens)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190430027) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190434081) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190421377) [haskellPackages.DescriptiveKeys](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.DescriptiveKeys) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190430228) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190420193) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190434137) [haskellPackages.align-audio](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.align-audio) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190434920) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190437763) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190437884) [haskellPackages.easytensor-vulkan](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.easytensor-vulkan) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190428304) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190426306) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190428625) [haskellPackages.harfbuzz-pure](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.harfbuzz-pure) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190421597) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190425098) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190436618) [haskellPackages.haskell-admin](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.haskell-admin) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190431791) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190430059) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190435855) [haskellPackages.hmatrix-nlopt](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hmatrix-nlopt) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190432249) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190427429) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190421622) [haskellPackages.hs-swisstable-hashtables-class](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hs-swisstable-hashtables-class) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190431132) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190425784) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190428436) [haskellPackages.hw-simd-cli](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.hw-simd-cli) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190426859) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190434719) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190432027) [haskellPackages.kmn-programming](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.kmn-programming) 
- [ ] [[:iphone::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/190428485) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190426386) [[:penguin::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/190430957) [haskellPackages.network-messagepack-rpc-websocket](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.network-messagepack-rpc-websocket) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190433858) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190434455) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190424957) [haskellPackages.rounded](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.rounded) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190421978) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190436722) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190421181) [haskellPackages.rounded-hw](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.rounded-hw) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190431105) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190421414) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190426503) [haskellPackages.shake-futhark](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.shake-futhark) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190436643) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190434723) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190430706) [haskellPackages.sound-collage](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.sound-collage) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190432799) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190425240) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190424340) [haskellPackages.unicode-names](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.unicode-names) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190434891) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190418740) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190423216) [haskellPackages.warp-quic](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.warp-quic) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190431301) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190420645) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190424172) [haskellPackages.xbattbar](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.xbattbar) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190434187) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190434826) [haskellPackages.xmonad-extras](https://hydra.nixos.org/eval/1780628?filter=haskellPackages.xmonad-extras) 
- [ ] [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190423364) [xmonad-with-packages](https://hydra.nixos.org/eval/1780628?filter=xmonad-with-packages) 
</details>

#### Top 50 broken packages, sorted by number of reverse dependencies
<details><summary>50 job(s) </summary>

[amazonka-core](https://packdeps.haskellers.com/reverse/amazonka-core) :arrow_heading_up: 185  
[gogol-core](https://packdeps.haskellers.com/reverse/gogol-core) :arrow_heading_up: 184  
[haskell98](https://packdeps.haskellers.com/reverse/haskell98) :arrow_heading_up: 153  
[enumerator](https://packdeps.haskellers.com/reverse/enumerator) :arrow_heading_up: 56  
[util](https://packdeps.haskellers.com/reverse/util) :arrow_heading_up: 49  
[derive](https://packdeps.haskellers.com/reverse/derive) :arrow_heading_up: 48  
[amazonka](https://packdeps.haskellers.com/reverse/amazonka) :arrow_heading_up: 43  
[accelerate](https://packdeps.haskellers.com/reverse/accelerate) :arrow_heading_up: 42  
[parseargs](https://packdeps.haskellers.com/reverse/parseargs) :arrow_heading_up: 42  
[MonadCatchIO-transformers](https://packdeps.haskellers.com/reverse/MonadCatchIO-transformers) :arrow_heading_up: 41  
[data-lens](https://packdeps.haskellers.com/reverse/data-lens) :arrow_heading_up: 33  
[rank1dynamic](https://packdeps.haskellers.com/reverse/rank1dynamic) :arrow_heading_up: 33  
[distributed-static](https://packdeps.haskellers.com/reverse/distributed-static) :arrow_heading_up: 31  
[language-ecmascript](https://packdeps.haskellers.com/reverse/language-ecmascript) :arrow_heading_up: 31  
[distributed-process](https://packdeps.haskellers.com/reverse/distributed-process) :arrow_heading_up: 30  
[iteratee](https://packdeps.haskellers.com/reverse/iteratee) :arrow_heading_up: 29  
[jmacro](https://packdeps.haskellers.com/reverse/jmacro) :arrow_heading_up: 29  
[mmsyn3](https://packdeps.haskellers.com/reverse/mmsyn3) :arrow_heading_up: 28  
[autodocodec-yaml](https://packdeps.haskellers.com/reverse/autodocodec-yaml) :arrow_heading_up: 27  
[crypto-numbers](https://packdeps.haskellers.com/reverse/crypto-numbers) :arrow_heading_up: 25  
[either-unwrap](https://packdeps.haskellers.com/reverse/either-unwrap) :arrow_heading_up: 25  
[sydtest](https://packdeps.haskellers.com/reverse/sydtest) :arrow_heading_up: 24  
[crypto-pubkey](https://packdeps.haskellers.com/reverse/crypto-pubkey) :arrow_heading_up: 22  
[haskelldb](https://packdeps.haskellers.com/reverse/haskelldb) :arrow_heading_up: 22  
[wxdirect](https://packdeps.haskellers.com/reverse/wxdirect) :arrow_heading_up: 22  
[alg](https://packdeps.haskellers.com/reverse/alg) :arrow_heading_up: 21  
[amazonka-s3](https://packdeps.haskellers.com/reverse/amazonka-s3) :arrow_heading_up: 21  
[mmsyn2](https://packdeps.haskellers.com/reverse/mmsyn2) :arrow_heading_up: 21  
[wxc](https://packdeps.haskellers.com/reverse/wxc) :arrow_heading_up: 21  
[biocore](https://packdeps.haskellers.com/reverse/biocore) :arrow_heading_up: 20  
[wxcore](https://packdeps.haskellers.com/reverse/wxcore) :arrow_heading_up: 20  
[attoparsec-enumerator](https://packdeps.haskellers.com/reverse/attoparsec-enumerator) :arrow_heading_up: 19  
[bytestring-show](https://packdeps.haskellers.com/reverse/bytestring-show) :arrow_heading_up: 19  
[fay](https://packdeps.haskellers.com/reverse/fay) :arrow_heading_up: 19  
[wx](https://packdeps.haskellers.com/reverse/wx) :arrow_heading_up: 19  
[asn1-data](https://packdeps.haskellers.com/reverse/asn1-data) :arrow_heading_up: 18  
[dbus-core](https://packdeps.haskellers.com/reverse/dbus-core) :arrow_heading_up: 18  
[gtksourceview2](https://packdeps.haskellers.com/reverse/gtksourceview2) :arrow_heading_up: 18  
[ukrainian-phonetics-basic](https://packdeps.haskellers.com/reverse/ukrainian-phonetics-basic) :arrow_heading_up: 18  
[HGamer3D-Data](https://packdeps.haskellers.com/reverse/HGamer3D-Data) :arrow_heading_up: 17  
[certificate](https://packdeps.haskellers.com/reverse/certificate) :arrow_heading_up: 17  
[dbus-client](https://packdeps.haskellers.com/reverse/dbus-client) :arrow_heading_up: 17  
[gconf](https://packdeps.haskellers.com/reverse/gconf) :arrow_heading_up: 17  
[gtk-serialized-event](https://packdeps.haskellers.com/reverse/gtk-serialized-event) :arrow_heading_up: 17  
[cuda](https://packdeps.haskellers.com/reverse/cuda) :arrow_heading_up: 16  
[happstack-jmacro](https://packdeps.haskellers.com/reverse/happstack-jmacro) :arrow_heading_up: 16  
[manatee-core](https://packdeps.haskellers.com/reverse/manatee-core) :arrow_heading_up: 16  
[monads-fd](https://packdeps.haskellers.com/reverse/monads-fd) :arrow_heading_up: 16  
[tls-extra](https://packdeps.haskellers.com/reverse/tls-extra) :arrow_heading_up: 16  
[ADPfusion](https://packdeps.haskellers.com/reverse/ADPfusion) :arrow_heading_up: 15  
</details>


*:arrow_heading_up:: The number of packages that depend (directly or indirectly) on this package (if any). If two numbers are shown the first (lower) number considers only packages which currently have enabled hydra jobs, i.e. are not marked broken. The second (higher) number considers all packages.*

*Report generated with [maintainers/scripts/haskell/hydra-report.hs](https://github.com/NixOS/nixpkgs/blob/haskell-updates/maintainers/scripts/haskell/hydra-report.sh)*

sternenseemann committed:
haskellPackages.xmonad: drop upstreamed Nix compat patch

We had a long standing downstream patch for XMonad allowing us to
override the ghc and xmessage binaries used. That has been upstreamed
and released in 0.17.1 and we can drop it!

* patch upstreamed in https://github.com/xmonad/xmonad/pull/409
* downstream patch adjusted in https://github.com/NixOS/nixpkgs/pull/184784

sternenseemann commented:
PR for xmonad at #191169.

github-actions[bot] committed:
Merge master into haskell-updates

arcz committed:
haskellPackages.miso: unbreak

cdepillabout committed:
Merge pull request #191343 from arcz/miso-unbreak

haskellPackages.miso: unbreak

cdepillabout committed:
haskellPackages: regenerate package set based on current config

This commit has been generated by maintainers/scripts/haskell/regenerate-hackage-packages.sh

sternenseemann commented:
Fix for weeder and HLS seems to downgrade megaparsec for GHC < 9.0 as it seems to have introduced a `base >= 4.15` constraint.

github-actions[bot] committed:
Merge master into haskell-updates

github-actions[bot] committed:
Merge master into haskell-updates

sternenseemann committed:
elm2nix: drop now released patches

sternenseemann committed:
haskell.packages.ghc{8107,884}.megaparsec: ignore base lower bound

maralorn commented:
I opened https://github.com/NixOS/nixpkgs/pull/191699 for the remaining build error for hedgewars on darwin. Hope we can clear that one up quickly because hls 1.8 has arrived on hackage and I am itching to get that one (including improving our hls setup).

github-actions[bot] committed:
Merge master into haskell-updates

maralorn committed:
Merge branch 'master' into haskell-updates

maralorn committed:
haskellPackages: mark builds failing on hydra as broken

This commit has been generated by maintainers/scripts/haskell/mark-broken.sh
